draft-ietf-quic-multipath.md

```diff
@@ -202,7 +202,7 @@ The same Path ID is used in both directions to
 address a path in the new multipath control frames,
 such as PATH_ABANDON {{path-abandon-frame}}, PATH_STANDBY {{path-standby-frame}}},
 PATH_AVAILABLE {{path-available-frame}} as well as MP_ACK {{mp-ack-frame}}.
-Further, connection IDs are issued per Path ID.
+Further, connection IDs are issued per Path ID (specified in {{mp-new-conn-id-frame}}).
```

Comment:
I think "using" rather "specified in" here would be more correct, or possibly "see" if one to be non-specific.

Reply:
Indeed. I updated.